### ISSUE TITLE
Can't install via composer

### ISSUE THREAD

**Issue by kkayacan** [j0k3r/graby#143]
Output: 
```
~/workspace $ composer require j0k3r/graby
Using version ^1.12 for j0k3r/graby
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for j0k3r/graby ^1.12 -> satisfiable by j0k3r/graby[1.12.0].
    - j0k3r/graby 1.12.0 requires htmlawed/htmlawed dev-master -> satisfiable by htmlawed/htmlawed[dev-master] but these conflict with your requirements or minimum-stability.


Installation failed, deleting ./composer.json
```

**Comment by tcitworld** [comment]
You may add to your composer.json file `minimum-stability: dev` https://getcomposer.org/doc/04-schema.md#minimum-stability

However, we could target `htmlawed 1.2`, since it looks like the same. https://github.com/kesar/HTMLawed

**Comment by kkayacan** [comment]
Thanks

**Comment by j0k3r** [comment]
I've released 1.12.1 which should fix the problem.
